ide/swing.validation/external/simplevalidation-1.14.1-license.txt

```diff
@@ -4,6 +4,7 @@ Version: 1.14.1
 Origin: GitHub
 License: Apache-2.0
 URL: https://github.com/timboudreau/simplevalidation/
+Files: simplevalidation-1.14.1.jar simplevalidation-swing-1.14.1.jar
```

Comment:
Instead of using two license files, lets try to only use one. This is to simplify using of the notice file.